SCClassLibrary/JITLib/GUI/ProxyMixer.sc

```diff
@@ -182,21 +182,25 @@ ProxyMixer : JITGui {
 	setEdButs { |isSmall = false|
 		(arGuis ++ krGuis).do { |pxgui|
 			pxgui.edBut.states_([
-					["ed", skin.fontColor, skin.background],
-					["ed", skin.fontColor, skin.onColor]])
-
-				.action_({ arg btn, mod;
-					if (mod.notNil and: { mod.isAlt }) {
-						NdefGui(pxgui.object);
-					} {
-						this.switchSize(2, isSmall);
-						editGui.object_(pxgui.object);
-						arGuis.do { |gui| gui.edBut.value_(0) };
-						krGuis.do { |gui| gui.edBut.value_(0) };
-						btn.value_(1);
-					};
-				});
-		};
+				["ed", skin.fontColor, skin.background],
+				["ed", skin.fontColor, skin.onColor]])
+
+			.action_({ arg btn, mod;
+				this.proxyspace.use {
+					protect {
```

Comment:
`protect` inside `use` is not needed, because `use` already does `protect`.

Reply:
ah thank, did not look up the implementation of .use